tests/bluetooth/bsim_bt/edtt_ble_test_app/tests_scripts/gap.test_list

```diff
@@ -20,7 +20,7 @@ GAP/CONN/ACEP/BV-01-C
 GAP/CONN/ACEP/BV-03-C
 GAP/CONN/ACEP/BV-04-C
 GAP/CONN/DCON/BV-01-C
-GAP/CONN/ENC
+#GAP/CONN/ENC
```

Comment:
Is this still intentional?

Reply:
It is intentional. There is no test GAP/CONN/ENC, so this line will probably be removed in the future